modules/caddyhttp/reverseproxy/fastcgi/fastcgi.go

```diff
@@ -54,6 +54,9 @@ type Transport struct {
 	// that 404s if the fastcgi path info is not found.
 	SplitPath []string `json:"split_path,omitempty"`
 
+	// Resolve symlink in root path
```

Comment:
This will be shown to users reading documentation, so explain what this is about.

Reply:
Added some background on opcache and detailed subdirective effects: I hope it's not to lengthy now

Reply:
Detailed is good :+1: